### PR TITLE
My Portfolio desktop version

### PR DESCRIPTION
**Pull request by Leeoasis**
In this pull request, the following is an outline of tasks performed:

- Add desktop version markup in index.html.
- Add desktop foreground and background images.
- Add the desktop version styling.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,18 +14,23 @@
...
+            <li class="menu-icon">
                 <img class="menu" src="static/images/Union.png" alt=" navigation" />
-            </button>
+            </li>
+            <li class="desktop-nav-list">Portfolio</li>
+            <li class="desktop-nav-list">About</li>
+            <li class="desktop-nav-list">Contact</li>
+          </ul>
 
         </nav>
```
Comment (victorokoroji) [inline index.html]: - [ ] Good luck aligning your header section. Kindly adjust the height and padding around the navigation elements to match the figma design and also the color around the header 👍🏽. 

Figma design 👇🏽 
image

Your design 👇🏽 
image

File: index.html [hunk 2]
```diff
@@ -14,18 +14,23 @@
         <nav class="nav-list">
             
             <a class="logo" href="#">My Logo</a>
```
Comment (victorokoroji) [inline index.html]: [OPTIONAL]

- I will recommend you replace this logo with your actual name, thereby personalizing your application 👍🏽 
image

File: index.html [hunk 3]
```diff
@@ -14,18 +14,23 @@
...
+            <li class="desktop-nav-list">About</li>
+            <li class="desktop-nav-list">Contact</li>
+          </ul>
 
         </nav>
 
 
     </header>
 
     <section class="container">
```
Comment (victorokoroji) [inline index.html:32]: - [ ] I will suggest you make sure that the background-image of the headline matches the figma design 👍🏽 

Your design 👇🏽 
image

FIgma design 👇🏽 
image

File: index.html [hunk 4]
```diff
@@ -14,18 +14,23 @@
...
-        <h1 class="greeting-text">I am Leslie Gudo <br />Glad to see you!</h1>
+        <h1 class="greeting-text">I am Leslie Glad to see you!</h1>
```
Comment (victorokoroji) [inline index.html]: - [ ] KIndly break the content of this headline text to match the figma design 👍🏽 

Figma design 👇🏽 
image

Your design 👇🏽 
image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @Leeoasis, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
-Good UI design ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -13,19 +13,24 @@
...
 
-            <button class="menu-button">
+            <li class="menu-icon">
                 <img class="menu" src="static/images/Union.png" alt=" navigation" />
-            </button>
+            </li>
+            <li class="desktop-nav-list">Portfolio</li>
+            <li class="desktop-nav-list">About</li>
+            <li class="desktop-nav-list">Contact</li>
+          </ul>
```
Comment (Nyame-Wolf) [inline index.html:25]: [OPTIONAL]I suggest you fix the mobile styling for your site. The headline section does not match the figma template. 
This is your design:
Screenshot 2023-01-20 at 09 12 27

this is the figma template:
Screenshot 2023-01-19 at 09 40 33

Remember, that it's important to show that given a template, you can create a matching design.

File: index.html [hunk 6]
```diff
@@ -248,8 +256,8 @@ <h2 class="contact-heading">Contact me</h2>
             <input class="contact-input" name="name" placeholder="Leslie" type="text" maxlength="30" required>
             <input class="contact-input" name="email" type="email" placeholder="leeegd99@gmail.com" required>
             <textarea class="textarea-input" name="textarea" cols="30" rows="9"
```
Comment (Nyame-Wolf) [inline index.html]: [OPTIONAL] I suggest you change the place holders to `name` and `youremail@gmail.com`. Generalising it this way ensures that you increase its visibility plus avoid confusion especially for screen readers.

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @Leeoasis  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:
- Linters are passing
- Used figms design correctly
- Used correct Github flow
## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**